firebase-project/src/views/postList.vue

```diff
@@ -131,12 +233,128 @@ export default {
 
     getAllPosts();
 
+    const goBackPost = () => {
+      router.push("/post");
+    };
+
+    const showCreateButton = computed(() => {
+      return lastVisiblePost.value !== null;
+    });
+
+    const handleAddComment = (postId) => {
+      showInput.value = postId;
+    };
+
+    const submitComment = async () => {
```

Comment:
can you move all these functionality to a composable api file @mukeshsin

Reply:
done